guides/doc-Provisioning_Guide/topics/proc_installing_ipv6_environment.adoc

```diff
@@ -0,0 +1,264 @@
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+hammer http-proxy create --name "Default proxy" --url http://myproxy.example.com:8888 --organizations "Default Organization" --locations "Default Location"
+hammer settings set --name content_default_http_proxy --value "Default proxy"
+----
+
+NOTE: The following can be a module/procedure. We need to drop a note that this is only relevant when Katello is installed if upstream.
+
+Import a manifest and synchronize {RHEL} 8 kickstart repository.
+
+== Creating new IPv6 Subnet
+
+To create new IPv6 Subnet, go to Infrastructure - Subnets and create new entry:
+
+Protocol: IPv6
+Network Address: valid IPv6 address
+Network Prefix: valid prefix
+Primary DNS Server: must be valid IPv6 address of a DNS server
+
+IPAM modes:
+
+EUI-64 - IPv6 address is calculated from host provisioning interface MAC address
+Internal DB - IPv6 address is selected from a sequence of all available addresses from the subnet
+None - IPv6 address must be entered manually
+
+Boot mode can be set both to DHCP or static, keep in mind that external DHCPv6 server is deployed and when this option is set to static a DHCPv6 service must be deployed on the network for the initial HTTP UEFI bootstrap.
+
+NOTE: Primary DNS server must be present becuase some older versions of {RHEL} 8 do not correctly interper DNS server coming from DHCPv6 response.
+
+To create a IPv6 subnet via hammer do:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer subnet create --name ipv6_subnet --network-type IPv6 --network 2001:db8:: --prefix 64 --tftp-id 1 --discovery-id 1 --httpboot-id 1 --template-id 1 --domains example.com
+----
+
+== Configuring Grub2 for HTTP UEFI
+
+NOTE: This step is a workaround for the following bugs in grub2 component:
++
+* https://bugzilla.redhat.com/show_bug.cgi?id=1732765
+* https://bugzilla.redhat.com/show_bug.cgi?id=1763216
+* https://bugzilla.redhat.com/show_bug.cgi?id=1616395
+
+Replace `/var/lib/tftpboot/grub2/grubx64.efi` deployed by the {foreman-installer} with a version from Fedora stable release. The file can be copied from installed Fedora EFI system or extracted from `grub2-efi` RPM package:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# wget https://download.fedoraproject.org/pub/fedora/linux/releases/31/Server/x86_64/os/Packages/g/grub2-efi-x64-2.02-100.fc31.x86_64.rpm
+# rpm2cpio grub2-efi-x64-2.02-100.fc31.x86_64.rpm | cpio -idmv
+./boot/efi/EFI/fedora/fonts
+./boot/efi/EFI/fedora/grubx64.efi
+./boot/grub2/grubenv
+./boot/loader/entries
+./etc/grub2-efi.cfg
+# cp ./boot/efi/EFI/fedora/grubx64.efi /var/lib/tftpboot/grub2/grubx64.efi
+----
+
+== Configuring external DHCPv6
+
+Both DHCPv6 and Router Advertisment services must be running on all provisioning networks. It can be installed on any host in the network, or on DHCP {SmartProxy}. {Project} currently does not manage DHCPv6 records, it is only used to bootstrap clients into installer.
```

Comment:
```suggestion
Both DHCPv6 and Router Advertisement services must be running on all provisioning networks. It can be installed on any host in the network, or on DHCP {SmartProxy}. {Project} currently does not manage DHCPv6 records, it is only used to bootstrap clients into the installer.
```

Reply:
Yeah thanks for these.